Science/src/temp_humidity/temp_humidity.cpp

```diff
@@ -1,23 +1,57 @@
 #include "temp_humidity.h"
 
-TempHumiditySensor::TempHumiditySensor() : sensor(&Wire, 0x44, 4) { }
+void TempHumiditySensor::readReg(){
+  Wire.beginTransmission(addr);
+  // Rewrote with overloaded method so that you don't need to use & on a literal
```

Comment:
So the reason we used `&Wire` instead of just `Wire` is that there are actually several different Wire objects, one for each I2C chip on the Teensy itself -- `Wire`, `Wire1`, and `Wire3`. Depending on which pins the electrical team decides to use, a different chip may be needed. 

In the past, we've found it tricky to switch back and forth (eg for testing or debugging) because we were using `Wire` all over the file. So we used a _reference_ to a wire object to allow passing in different wire objects and be sure they all changed at once. Also acceptable is `#define WIRE Wire` and then that can be changed as well.